contrib/jemalloc-cmake/include_linux_aarch64/jemalloc/internal/jemalloc_internal_defs.h.in

```diff
@@ -279,9 +279,11 @@
  *                                 MADV_FREE, though typically with higher
  *                                 system overhead.
  */
+#if defined(__linux__)
 #define JEMALLOC_PURGE_MADVISE_FREE
+#else
 #define JEMALLOC_PURGE_MADVISE_DONTNEED
-#define JEMALLOC_PURGE_MADVISE_DONTNEED_ZEROS
```

Comment:
FWIW `JEMALLOC_PURGE_MADVISE_DONTNEED_ZEROS` should be disable for non-linux

Reply:
>FWIW JEMALLOC_PURGE_MADVISE_DONTNEED_ZEROS should be disable for non-linux (regardless this PR)

And there are some other issues with shipped header config, I guess it is better to fix this separately (linux is ok, at least freebsd has some issues)